tests_e2e/tests/scripts/agent-service

```diff
@@ -0,0 +1,87 @@
+#!/usr/bin/env bash
+
+# Microsoft Azure Linux Agent
+#
+# Copyright 2018 Microsoft Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+set -euo pipefail
+
+#
+# The service name is walinuxagent in Ubuntu/debian and waagent elsewhere
+#
+
+usage() (
+    echo "Usage: service-command -c|--cmd name"
```

Comment:
I'd suggest making the command a positional argument, rather than a keyword argument, makes the script invocation a little simpler: "agent-service stop"